factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/PgConfigurationProperties.java

```diff
@@ -48,17 +48,10 @@
   @Autowired private ApplicationContext appContext;
 
   /** defines the number of Facts being retrieved with one Page Query for PageStrategy.PAGED */
-  int pageSize = 1000;
-
-  /** The capacity of the queue for PageStrategy.QUEUED */
-  int queueSize = 1000;
-
+  int pageSize = 50;
```

Comment:
The comment needs to be adapted (if I understand correctly, the pageSize will be also used for FETCHING)